core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -147,6 +148,15 @@ private[spark] class SparkHadoopUtil extends Logging {
     }
   }
 
+  /**
+   *
```

Comment:
Maybe remove this line or add some description

Reply:
done